packages/transport-http/src/http-request.js

```diff
@@ -121,6 +134,9 @@ export async function httpRequest({
           })
         })
         .catch(e => {
+          if (e.message === "Failed to fetch") {
```

Comment:
QQ: Is this for CORS errors? If so, is this the message for all CORS error responses? Or is this just for the Flow REST APIs? 🤔

Reply:
Yeah this is an error thrown by the fetch API, not the access node.

It's for network errors that cause fetch to fail.  Obviously there could be other conditions other than CORS (i.e. domain name not found, etc.). But it all sort of fits into the category of something being wrong with the provided endpoint so the current error message is quite fitting.

However - this appears to me chrome specific after a bit of testing so I will try to get something multiplatform